tasks/main.yml

```diff
@@ -1,8 +1,16 @@
 ---
 
+- name: Fail if os family is not redhat or debian or windows
+  fail:
+    msg: Current role does not support on {{ ansible_os_family | lower }}, It only supports CentOS/RedHat, Debian and Windows hosts.
+  when: (ansible_os_family | lower) not in ['redhat', 'debian', 'windows']
```

Comment:
Do we support windows for regional proxy? I guess this is long ago we stopped running on window.

Reply:
As far as I know, few customers still use Windows.